templates/zerver/integrations.html

```diff
@@ -553,6 +559,31 @@
       <img class="screenshot" src="/static/images/integrations/bitbucket/002.png" />
     </div>
 
+    <div id="bitbucket2" class="integration-instructions">
+
+     <p>Zulip supports both Git and Mercurial notifications from Bitbucket.</p>
+
+     <p>First, create the stream you'd like to use for Bitbucket notifications,
+     and subscribe all interested parties to this stream. The integration will
+     use the default stream <code>bitbucket</code> if no stream is supplied in the
+     hook; you still need to create the stream even if you are using this
+     default.</p>
+
+     <p><code>{{ external_api_uri }}/v1/external/bitbucket2?api_key=abcdefgh&amp;stream=bitbucket</code></p>
```

Comment:
It would be great to label this chunk of code in some way, such as by prefacing it with:

> The URL you create will be in the following format: